From 1867dd08ec2730675e28b870ad97523a5bc88b15 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Thu, 10 Nov 2016 15:53:03 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399: modify gpu opp table for evb
 board

margin 25mV-50mV, stress test:
1. antutu-3d, use governor simpleondemand
2. webgl, fish number 50, sweep frequency
3. glmark2, run texture and shadow, sweep frequency

Change-Id: Ia2682610e948df7df2ad190ac3a28b2dad464cb3
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
index 36cfd18a0fdc..335a016b1713 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
@@ -226,7 +226,7 @@
 	};
 	opp@500000000 {
 		opp-hz = /bits/ 64 <500000000>;
-		opp-microvolt = <850000>;
+		opp-microvolt = <825000>;
 	};
 	opp@600000000 {
 		opp-hz = /bits/ 64 <600000000>;
@@ -234,7 +234,7 @@
 	};
 	opp@800000000 {
 		opp-hz = /bits/ 64 <800000000>;
-		opp-microvolt = <1000000>;
+		opp-microvolt = <1050000>;
 	};
 };
 
-- 
2.35.3

